src/containers/Header/index.js

```diff
@@ -31,7 +31,7 @@ export function Header(props) {
 		<header className={`application-header ${headerProperties ? 'header-props-open' : ""}`}>
 			{(props.app.props.hasSidebar || typeof props.app.props.hasSidebar == 'undefined') ?
 				<>
-					{windowDimensions.width < 800 &&
+					{windowDimensions.width < 769 &&
```

Comment:
@Aringoaway Out of curiosity - on line 46 you have this condition: 

https://github.com/TeskaLabs/asab-webui/blob/ab29a72cd4cb2029323e05bb785eb8a305d6eef1/src/containers/Header/index.js#L46

Why those numbers differ?